src/protocols/pubsub/trie.c

```diff
@@ -299,7 +298,6 @@ int nn_trie_subscribe (struct nn_trie *self, const uint8_t *data, size_t size)
     memmove (ch->prefix, ch->prefix + pos + 1, ch->prefix_len);
     ch = nn_node_compact (ch);
     *nn_node_child (*node, 0) = ch;
-    pos = (*node)->prefix_len;
```

Comment:
I haven't actually tried to understand the code.  Its probably dead from an earlier implementation, but I had nothing to do with it.